src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -65,7 +76,7 @@ internal abstract class JsonPropertyInfo
 
         internal bool IsNullableType { get; private set; }
 
-        public PropertyInfo PropertyInfo { get; private set; }
+        internal PropertyInfo PropertyInfo { get; private set; }
```

Comment:
Does it make sense to have internal properties on an internal type?

Reply:
Normally I'd make these public. The reason these are internal is that this class was at one point going to be made public to expose metadata for the property. That may or may not be the case going forward.